Mathlib/Algebra/Category/ModuleCat/Ulift.lean

```diff
@@ -0,0 +1,96 @@
+/-
+Copyright (c) 2025 Nailin Guan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nailin Guan
+-/
+module
+
+public import Mathlib.Algebra.Category.ModuleCat.Injective
+public import Mathlib.Algebra.Category.ModuleCat.Projective
+public import Mathlib.Algebra.Homology.ShortComplex.ModuleCat
+public import Mathlib.CategoryTheory.Linear.LinearFunctor
+public import Mathlib.CategoryTheory.Preadditive.Injective.Preserves
+public import Mathlib.CategoryTheory.Preadditive.Projective.Preserves
+
+/-!
+
+# Ulift Functor of ModuleCat
+
+In this file, we define the obvious functor `ModuleCat.{v} R ⥤ ModuleCat.{max v v'} R` and prove
+it is exact, fully faithful and preverves projective and injective objects.
+
+-/
+
+@[expose] public section
+
+universe v' v u
+
+variable (R : Type u)
+
+open CategoryTheory
+
+namespace ModuleCat
+
+section Ring
+
+variable [Ring R]
+
+/-- Universe lift functor for `R`-module. -/
+@[simps obj map]
```

Comment:
```suggestion
@[simps obj map, pp_with_univ]
```
This forces the pretty-printer to display the involved universes in the infoview for this declaration.